### DISCUSSION TITLE
Stopping Power Ratio for spherical chambers in EGSnrc

### DISCUSSION THREAD

**Discussion by karenrex1023** [nrc-cnrc/EGSnrc#1154] [Questions]
How would you suggest computing the stopping power ratio between the air and chamber wall for spherical ionization chambers. I assume that SPPRZ is only for cylindrical geometries, but I am not sure if there is any egs code I can use to calculate the stopping power ratio for an A3 with a Co-60 spectrum?

**Comment by rtownson** [comment]
I'm not sure your final goal, but you might consider doing a full chamber simulation (e.g. using egs_chamber and your chamber specifications), and looking at correction factors through that instead of stopping power ratios.

I believe that if you have charged particle equilibrium, then the stopping power ratios should be independent of the geometry. So you can use cylinders instead of spheres. You could test this by adjusting the radius of the cylinders. But there are some subtleties in being sure that charged particle equilibrium is indeed valid. Even with SPRRZnrc, you still have to carefully choose AE to represent the delta appropriately for your chamber, and this requires an understanding of cavity theory. So it's better to just do the proper chamber model and full simulations.
